bundler.d/test.rb

```diff
@@ -18,7 +18,7 @@
   gem 'rubocop-rails', '~> 2.4.2'
   gem 'poltergeist', '>= 1.18.0', :require => false
   gem 'selenium-webdriver', :require => false
-  gem 'shoulda-matchers', '~> 4.0'
+  gem 'shoulda-matchers', '>= 4.0', '< 4.4'
```

Comment:
Is it reasonable to expect they are going to fix it in the next release and just exclude the bad release?
```suggestion
  gem 'shoulda-matchers', '~> 4.0', '!= 4.4.0'
```

Reply:
From the linked commit, it suggests it started to depend on the Zeitwerk loader. https://github.com/theforeman/foreman/pull/7843 did start the effort to move Foreman to use it, but that's still a draft.

Reply:
reading the issue: https://github.com/thoughtbot/shoulda-matchers/issues/1332  its unclear at this point.  The change that broke it almost seems like it was intentional, but i don't know that a developer has commented on the issue.